ex/loqui/lib/loqui/client.ex

```diff
@@ -0,0 +1,358 @@
+defmodule Loqui.Client do
+  defmodule State do
```

Comment:
Check out the differences between 16c7cb0b3f5ed43369b159f94d9977428f46bfc1 and ab60dd8ff0ccff11a86e87949015553b26f2e283 for an example of how having a separate module for state cleans up code.